site-src/guides/api-overview.md

```diff
@@ -0,0 +1,129 @@
+## API overview
+
+Prior to the AdminNetworkPolicy API there was no native tooling for Cluster Admins
+to apply security rules in a cluster-wide manner, and in some cases Network Policies
+were being incorrectly used to do so, creating a complex web of objects to be
+maintained.
+
+With the advent of the AdminNetworkPolicy API Cluster Admins will now have the 
+ability to apply policy on in-cluster workloads with only a few simple policy 
+objects that can be applied globally.
+
+## Roles and personas
+
+In this documentation we refer to three primary personas:
+
+- Application Developer
+- Namespace Administrator
+- Cluster Administrator
+
+## Resource Model 
+
+There are two main objects in the AdminNetworkPolicy API resource model
+
+- **AdminNetworkPolicy (ANP)**
+
+- **BaselineAdminNetworkPolicy (BANP)**
+
+## General Notes
+
+- Any well defined AdminNetworkPolicy rules should
```

Comment:
*rule

Reply:
I'm referring to the various and often multiple "rules" defined in an AdminNetworkPolicy Here